src/bin/rdcore/rootmap.rs

```diff
@@ -307,3 +307,30 @@ fn write_boot_uuid_grub2_dropin<P: AsRef<Path>>(uuid: &str, p: P) -> Result<()>
     std::fs::write(p, format!("set BOOT_UUID=\"{uuid}\"\n"))
         .with_context(|| format!("writing {}", p.display()))
 }
+
+fn get_multipath_configured() -> Result<Vec<String>> {
```

Comment:
"configured" is kind of generic. Maybe let's just rename to mention exactly what we are returning?

```suggestion
fn get_multipath_kargs() -> Result<Vec<String>> {
```